CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 
 - [#2893](https://github.com/thanos-io/thanos/pull/2893) Store: Rename metric `thanos_bucket_store_cached_postings_compression_time_seconds` to `thanos_bucket_store_cached_postings_compression_time_seconds_total`.
 - [#2915](https://github.com/thanos-io/thanos/pull/2915) Receive,Ruler: Enable TSDB directory locking by default. Add a new flag (`--tsdb.no-lockfile`) to override behavior.
+- [#2935](https://github.com/thanos-io/thanos/pull/2935) Replace sync/atomic with uber-go/atomic
```

Comment:
This is not relevant to user, I think, so we can remove.

We did not have any known bug due to old atomic (:

Reply:
Fair enough, addressed in 6dd75ce